example/webpack.prod.js

```diff
@@ -1,6 +1,6 @@
 const webpack = require('webpack');
 const merge = require('webpack-merge');
-const CleanWebpackPlugin = require('clean-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
```

Comment:
Usage has changed, and `dist` is default dir.
https://github.com/johnagan/clean-webpack-plugin#usage